polaris-react/src/components/Tile/Tile.scss

```diff
@@ -1,7 +1,37 @@
 @import '../../styles/common';
 
 .Tile {
+  --pc-tile-xs: 6;
+  --pc-tile-sm: var(--pc-tile-xs);
+  --pc-tile-md: var(--pc-tile-sm);
+  --pc-tile-lg: var(--pc-tile-md);
+  --pc-tile-xl: var(--pc-tile-lg);
+  --pc-tile-gap-xs: var(--p-space-4);
+  --pc-tile-gap-sm: var(--pc-tile-gap-xs);
+  --pc-tile-gap-md: var(--pc-tile-gap-sm);
+  --pc-tile-gap-lg: var(--pc-tile-gap-md);
+  --pc-tile-gap-xl: var(--pc-tile-gap-lg);
```

Comment:
Are the tokens supposed to go from `var(--pc-tile-gap-xs)` to `var(--pc-tile-gap-lg)` or should `--pc-tile-gap-xs` use `var(--pc-tile-gap-xs)` so that the tshirt sizes matches and we go up to `var(--pc-tile-gap-xl)`?

Reply:
I used the `Grid` and `Columns` components as a reference for this, but from my understanding it allows a cascading effect so if the user doesn't select a value for the specific breakpoint it will use the initial value throughout considering mobile first. For example if the user only inputs `gap={{md: '32'}}`

The styling would look like this:
```jsx
--pc-tile-gap-xs: var(--p-space-4);
--pc-tile-gap-sm: var(--p-space-4);
--pc-tile-gap-md: var(--p-space-32);
--pc-tile-gap-lg: var(--p-space-32);
--pc-tile-gap-xl: var(--p-space-32);
```